storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -650,7 +666,7 @@ func addResourceTargetInfo(resource pcommon.Resource, settings Settings, earlies
 		// Do not pass identifying attributes as ignoreAttrs below.
 		identifyingAttrs = nil
 	}
-	labels, err := createAttributes(resource, attributes, scope{}, settings, identifyingAttrs, false, prompb.MetricMetadata{}, model.MetricNameLabel, name)
+	labels, err := createAttributes(resource, attributes, scope{}, settings, identifyingAttrs, false, prompb.MetricMetadata{}, 0, model.MetricNameLabel, name)
```

Comment:
[Nit] Could we use the corresponding constant for improved readability, and not rely on the constant's value?

```suggestion
	labels, err := createAttributes(resource, attributes, scope{}, settings, identifyingAttrs, false, prompb.MetricMetadata{}, pmetric.AggregationTemporalityUnspecified, model.MetricNameLabel, name)
```